Representable/representable
# fix video + change contiguity check

**changes**
- fixed video embed on map page
- changed contiguity check to a warning rather than a block
- small changes to formatting of video button

**to test**
- python manage.py runserver
- navigate to entry page mapping portion
- check video is visible
- check contiguity check does not block but instead warns the user

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/105804441-eab35180-5f43-11eb-9035-6bd964eac3f4.png)
